README.md

```diff
@@ -69,6 +69,42 @@ Get all tags for a model
 model.tags
 ```
 
+### Training a Named Entity Extractor
+
+Load a MITIE NER model into a trainer
+
+```ruby
+trainer = Mitie::Trainers::NER.new("total_word_feature_extractor.dat")
+```
+
+Create training instances to train against
+
+```ruby
+sample = Mitie::TrainingInstances::NER.new(["My", "name", "is", "Davis", "King", "and", "I", "work", "for", "MIT", "."])
+sample.add_entity((3...5), "person") # Davis King
+sample.add_entity((9...10), "org") # MIT
```

Comment:
Should be able to remove parentheses around ranges in examples and tests: `(9...10)` -> `9...10`